fitbenchmarking/fitbenchmarking
# Support for bumps 0.9.0 (remove mp, add scipy.leastsq)

#### Description of Work

Fixes #1038.

I've:
- pinned bumps to >=0.9.0
- removed support for `mp`, and added `scipy-leastsq` as an option
- changed the docs to describe what these options do
- updated output files for systests


#### Testing Instructions

1. Check the new bumps minimizers work

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
